hayeon/programmers-1/나누어-떨어지는-숫자.js

```diff
@@ -0,0 +1,31 @@
+function solution(arr, divisor) {
+  let newArr = [];
+
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] % divisor === 0) {
+      newArr.push(arr[i]);
+      newArr.sort((a, b) => a - b);
+    }
+  }
+  if (newArr.length === 0) {
+    newArr.push(-1);
+  }
+  return newArr;
+}
```

Comment:
```suggestion
function solution(arr, divisor) {
  let newArr = [];
    
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] % divisor === 0) {
      newArr.push(arr[i]);
    }
  }
  
  newArr.sort((a, b) => a - b);
  
  if (newArr.length === 0) {
    return [-1]
  }
  return newArr;
}
```
- 고생하셨습니다 👍 피드백을 드리자면 for문을 돌릴 때 마다 sort를 실행하는건 비효율적인 풀이인 것 같습니다. 모든 for문을 돌리고 sort 딱 1번만 실행해도 원하는 결과값을 도출할 수 있습니다.
- 또한 이건 코딩스타일 차이인데 단순히 바로 return [-1]을 해줘도 괜찮다는 생각이 들었습니다  🙆‍♂️

Reply:
오옷 그렇군요!!!! 더 간단하네요 return [-1] 을 할 생각을 아예 못한....감사합니다👍🏻👍🏻👍🏻👍🏻